Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -401,6 +399,45 @@ def get_analysis_metadata_command(intezer_api: IntezerApi, args: dict) -> Comman
     )
 
 
+def get_analysis_iocs_command(intezer_api: IntezerApi, args: dict) -> CommandResults:
+    analysis_id = args.get('analysis_id')
+
+    try:
+        analysis = FileAnalysis.from_analysis_id(analysis_id, api=intezer_api)
+    except HTTPError as error:
+        if error.response.status_code == HTTPStatus.CONFLICT:
+            return _get_analysis_running_result(analysis_id=str(analysis_id))
+        raise
+
+    if not analysis:
+        return _get_missing_analysis_result(analysis_id=str(analysis_id))
+
+    iocs = analysis.iocs
+    readable_output = ''
+    if iocs:
+        network_iocs = iocs.get('network')
+        if network_iocs:
+            readable_output += tableToMarkdown('Network IOCs', network_iocs)
+        files_iocs = iocs.get('files')
+        if files_iocs:
+            readable_output += tableToMarkdown('Files IOCs', files_iocs)
```

Comment:
Would be better to use Walrus operator:
```suggestion
    if iocs:
        if network_iocs := iocs.get('network'):
            readable_output += tableToMarkdown('Network IOCs', network_iocs)
        if files_iocs := iocs.get('files'):
            readable_output += tableToMarkdown('Files IOCs', files_iocs)
```